tests/postgres_tests/test_json.py

```diff
@@ -162,6 +162,33 @@ def test_ordering_by_transform(self):
         query = JSONModel.objects.filter(field__name__isnull=False).order_by('field__ord')
         self.assertSequenceEqual(query, [objs[4], objs[2], objs[3], objs[1], objs[0]])
 
+    def test_order_by_and_group_by_with_list_params(self):
+        objs = [
+            JSONModel.objects.create(field={
+                'translations': [
+                    {'title': 'Ladies and gentleman'},
+                    {'title': 'Dames en heren'}
+                ]
+            }),
+            JSONModel.objects.create(field={
+                'translations': [
+                    {'title': 'Apples and pears'},
+                    {'title': 'Appels en peren'}
+                ]
+            }),
+        ]
```

Comment:
IMO we don't need to create new objects (`objs = [...]`), we should be able to change queries and reuse existing ones, e.g.
```python
def test_ordering_grouping_by_key_transform(self):
    self.assertSequenceEqual(
        JSONModel.objects.filter(field__d__0__isnull=False).order_by('field__d__0'),
        [self.objs[8]],
    )
    qs = JSONModel.objects.filter(field__isnull=False).values('field__d__0').annotate(
        count=Count('field__d__0'),
    ).order_by('count')
    self.assertQuerysetEqual(qs, [1, 10], operator.itemgetter('count'))
```